frontend/src/components/common/rich-text-editor/RichTextEditor.module.scss

```diff
@@ -308,4 +308,49 @@
       text-align: justify;
     }
   }
+
+  // Typography styles to match email theme
+  // Changes here should also be made in email-theme.mustache to update the actual email theme
+  // These styles are repeated in EmailPreviewBlock.module.scss due to differences in DOM structure
+  color: #000000;
+  font-size: 14px;
+
+  * {
+    font-family: Arial, Helvetica, sans-serif;
+  }
+
+  h1 {
+    font-size: 28px !important;
```

Comment:
Generally, we should avoid `!important` unless we really do not have a way to specifically target the component you want to style. Overuse of `!important` would make it hard to reason about what styles will apply since the usual CSS order of precedence are no longer respected. 

In this case, I think `!important` can be avoided by moving the CSS blocks into `.editor`.

In addition, the styles can be extracted into a separate file and shared using `@extend` and placeholder selectors (similar to our typography.scss). This ensure that there's only one place where the styles need to be modified